ci/deps/travis-36-doc.yaml

```diff
@@ -33,8 +33,8 @@ dependencies:
   - pytz
   - scipy
   - seaborn
-  # recursion error with sphinx 2.1.0. https://github.com/pandas-dev/pandas/issues/26723
-  - sphinx==2.0.1
+  # some styling is broken with sphinx >= 2 (https://github.com/pandas-dev/pandas/issues/26058)
+  - sphinx==1.8.5
```

Comment:
In conda files the version is specified with just one equal (`==` is for pip, a bit confusing)

You'll also have to call `./scripts/generate_pip_deps_from_conda.py` to update `requirements_dev.txt`, which is synched from `environment.yml`

Reply:
Ah, thanks!

Reply:
you also need to remove one equal here, other than that looks good

Reply:
Just wondering: does it actually matter? (it was with double equals before as well)

Reply:
not sure, but we've got a function to convert them from one to two when synching, if we can have two may be better to leave two everywhere and simplify the synching